action.yml

```diff
@@ -8,6 +8,9 @@ inputs:
   append:
     description: "Indicate if new comment messages should be appended to previous comment message"
     required: false
+  replace:
+    description: "Indicate if previous comment should be removed before creating a new comment"
+    required: false
```

Comment:
`recreate` seems to be a better name for this behavior, since default behavior also replaces inside of a comment.

Reply:
yeah, that's better. just updated.